packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -147,20 +167,191 @@ const SigninContainer = ({
 
   const [beginSignin] = useMutation<BeginSigninResponse>(BEGIN_SIGNIN_MUTATION);
 
+  const [credentialStatus] = useMutation<CredentialStatusResponse>(
+    CREDENTIAL_STATUS_MUTATION
+  );
+
+  const [passwordChangeStart] = useMutation<PasswordChangeStartResponse>(
+    PASSWORD_CHANGE_START_MUTATION
+  );
+
+  const [passwordChangeFinish] = useMutation<PasswordChangeFinishResponse>(
+    PASSWORD_CHANGE_FINISH_MUTATION
+  );
+
+  const [getWrappedKeys] = useMutation<GetAccountKeysResponse>(
+    GET_ACCOUNT_KEYS_MUTATION
+  );
+
   const beginSigninHandler: BeginSigninHandler = useCallback(
     async (email: string, password: string) => {
       // TODO in oauth ticket
       // const service = integration.getService();
       const options = {
         verificationMethod: VerificationMethods.EMAIL_OTP,
       };
+
+      function handleGqlError(error: any) {
```

Comment:
Not for this PR, but I've seen the `handleGqlError` in several different components, slightly different each time. It seems like something we could refactor more generally.